FO: Import CSV error 1.7.4.x

peladillas opened the pull request:
I have tried with several sizes of files and the error persists. I always get the same error "Uncaught typeError: Can not read property 'totalCount' of null" in import.js:195 in the middle of the import.

<!-- Thank you for contributing to the PrestaShop project! 

Please take the time to edit the "Answers" rows with the necessary information: -->

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Please be specific when describing the PR. <br/> Every detail helps: versions, browser/server configuration, specific module/theme, etc.
| Type?         | bug fix / improvement / new feature
| Category?     | See [the Category list](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message#Howtowriteacommitmessage-Category), i.e.: BO
| BC breaks?    | Does it break backward compatibility? yes/no
| Deprecations? | Does it deprecate an existing feature? yes/no
| Fixed ticket? | (optional) If this PR fixes a [Forge](http://forge.prestashop.com/) ticket, please add its complete Forge URL.
| How to test?  | Please indicate how to best verify that this PR is correct.

<!-- Click the form's "Preview button" to make sure the table is functional in GitHub. Thank you! -->

#### Important guidelines

* Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
* Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/prestashop/prestashop/9338)
<!-- Reviewable:end -->

jocel1 committed:
with should also check cart rules which begin before the current day, and finish after the current day

jocel1 committed:
fix cart rule check tests

kpodemski committed:
CO: ability to use widget block from specific hook

jocel1 committed:
optimize cart_rule table

jocel1 committed:
add namespace

lucasRolff committed:
CO: Fix bug with friendly URLs and Media Servers

Commit:
add combinations in cart summary

Commit:
indentation

PierreRambaud committed:
Use spaces instead tabs

Quetzacoalt91 committed:
Limit SQL results when tab is duplicated

Quetzacoalt91 committed:
Update Controller name for link generation to module catalog

PierreRambaud committed:
Merge pull request #9252 from marionf/boom-5470

add combinations in cart summary

mickaelandrieu committed:
Added .htaccess to var folder

toutantic committed:
Merge pull request #9268 from mickaelandrieu/bug/added-htaccess-file-to-var

Added .htaccess to var folder

toutantic committed:
Update ps_themecusto to version 1.0.6

PierreRambaud committed:
Make htaccess compatible apache 2.2 and 2.4

PierreRambaud committed:
Merge pull request #9101 from jocel1/fix-have-cart-rule-today

Fix have cart rule today

Commit:
// Catch more exceptions during install

PierreRambaud committed:
Merge pull request #9237 from lucasRolff/fix-seo-friendly-media-server-urls-1.7

Fixed bug with friendly URLs and Media Servers

toutantic committed:
Merge pull request #9269 from toutantic/Update-ps_themecusto

Update ps_themecusto to version 1.0.6

PierreRambaud committed:
Merge pull request #9261 from Quetzacoalt91/update-link-catalog-modules

Update Controller name for link generation to modules catalog

toutantic committed:
Merge pull request #9270 from PierreRambaud/apache/htaccess

Apache 2.4 configuration

Quetzacoalt91 committed:
Merge pull request #9220 from kpodemski/patch-19

Ability to use widget block from specific hook

Pedrock committed:
BO: Fix detection of APCu on Performance page

mickaelandrieu committed:
Tried to fix develop testsuite

PierreRambaud committed:
Merge pull request #9258 from Quetzacoalt91/upgrade-sql-1.7.0.0

Limit subquery results in upgrade sql 1.7.0.0

prestamodule committed:
CO: Fix Cart::isVirtualCart() method when cart is empty

tomlev committed:
fix free shipping display on cart

hadjedjvincent committed:
CO: Add Cart::hasProducts() method and use it into Cart:: isVirtualCart() (thanks @jocel1)

hadjedjvincent committed:
CO: Prefer using « INNER JOIN » instead of « JOIN »

hadjedjvincent committed:
CO: Do not test Cart::hasProducts() into Cart::hasRealProducts()

prestamodule committed:
BO: Fix display when a module uses $this->bootstrap = false;

tomlev committed:
fix error while duplicating a product when catalog specific price rules are stored

Quetzacoalt91 committed:
Merge pull request #9298 from mickaelandrieu/fix/remove-mbo-from-tests

Fix/remove mbo from tests

eternoendless committed:
Build assets

jocel1 committed:
Reduce the number of ajax calls if the fixtures are not too larges and if there's enough memory available

jocel1 committed:
post review fixes

mickaelandrieu committed:
Merge pull request #9297 from prestamodule/patch-105

Fix display when a module uses $this->bootstrap = false;

mickaelandrieu committed:
Merge pull request #9295 from tomlev/BOOM-5665

Fix free shipping display on cart

prestamodule committed:
CO: Do not apply wrapping fees if cart is empty

Commit:
Catch every exceptions on ajax process steps

hadjedjvincent committed:
CO: Prefer using hasProducts and hasRealProducts (perfs improvements)

Quetzacoalt91 committed:
Apply shop context in configuration

eternoendless committed:
Merge pull request #9288 from prestamodule/patch-104

Fix Cart::isVirtualCart() method when cart is empty

jocel1 committed:
most review fixes

tomlev committed:
fix specific price duplicate

eternoendless committed:
Merge pull request #9271 from rGaillard/fix_user_assistance

Catch more exceptions during install & display the error intead of a JS error

tomlev committed:
fix template error when trying to display not visible category

eternoendless committed:
Merge pull request #9299 from tomlev/BOOM-4001

Fix error while duplicating a product when catalog specific price rules are stored

PierreRambaud committed:
Merge pull request #9315 from tomlev/BOOM-5678-2

Fix not visible category display

eternoendless committed:
Merge pull request #9311 from Quetzacoalt91/configuration-shop

Apply shop context in configuration

eternoendless committed:
Merge pull request #9302 from jocel1/reduce-install-ajax-calls

Reduce the number of ajax calls if the fixtures are not too larges and if there's enough memory available

PierreRambaud committed:
Do not generate file path if file already exists

eternoendless committed:
// Changelog 1.7.4.1

PierreRambaud committed:
Translator function must select the right source

Currently the selected source is the extend module.
Always use $smarty->resource->name to retrieve the template name.

PierreRambaud committed:
Useless ret

PierreRambaud committed:
Improve reading style

mickaelandrieu committed:
Fixed Product form when stock management is disabled

PierreRambaud committed:
Use latest version of PHPSQLParser

This one hasn't been updated since 2014. Use composer repository
instead tools/parser_sql.

PierreRambaud committed:
Add missing pushState when switching combination

mickaelandrieu commented:
Hello @peladillas, there is no commit from you here.

Also, filling the contribution table is required to get a review from the team, thanks for your understanding.

Commit:
CO: fix bug extra space: htaccess directive order

PierreRambaud committed:
Merge pull request #9279 from Pedrock/fix-apcu-detection

Fix detection of APCu on Performance page

PierreRambaud committed:
Merge pull request #9322 from PierreRambaud/fix/boom-5915

Backward compatibility of  $this->isCached() in modules

matks committed:
Merge pull request #9339 from prestarocket/fix_apache_order_deny

CO: fix bug extra space: htaccess directive order

PierreRambaud committed:
Use document.title everywhere like in admin-theme.js

eternoendless committed:
Merge pull request #9327 from mickaelandrieu/BOOM-6000-shop-parameters

Fixed Product form when stock management is disabled

PierreRambaud committed:
One more test

eternoendless committed:
Merge pull request #9326 from PierreRambaud/fix/boom-5983

Module translations aren't working when using Smarty Extend into TPL

eternoendless committed:
Merge pull request #9330 from PierreRambaud/fix/boom-5954

It is impossible to create a SQL query "SELECT"  - SQL Manager page

PierreRambaud committed:
Can't install prestashop in dev mode with open_basedir

When composer.json file is missing and the open_basedir directive is
set we are not able to install Prestashop. More, there can be more
issues because the getProjectDir is wrong.

Commit:
Fixed a misnamed variable

Fixed a misnamed variable in Product::getIdProductAttributeByIdAttributes()

Quetzacoalt91 committed:
Merge pull request #9335 from PierreRambaud/fix/boom-5929

Url does not update when changing product attributes

Quetzacoalt91 committed:
Merge pull request #9345 from PierreRambaud/fix/open_basedir

Can't install prestashop in dev mode with open_basedir

eternoendless committed:
Merge pull request #9348 from jbroutier/1.7.4.x

Fixed a misnamed variable

123monsite-regis committed:
Set correct Employee Css after update (#9354)

* BO : Set correct Employee Css after update

* Create 1.7.4.2.sql

* Update 1.7.4.1.sql

* Update 1.7.4.1.sql

* Update 1.7.4.1.sql

eternoendless committed:
Update native modules

PierreRambaud committed:
Allow configuration to save html data

UpdateValue already allows data to be saved with html, when using
Configuration::set() we need to add a parameter to allow it again.

PierreRambaud committed:
Use options parameters instead of single one html

Follow @Quetzacoalt91 recommandations ;)

PierreRambaud committed:
Cast to bool to be user everything will be ok

PierreRambaud committed:
Missing array in signature

PierreRambaud committed:
Missing test update for phpunit

Quetzacoalt91 committed:
Merge pull request #9359 from PierreRambaud/fix/boom-6007

Allow configuration to save html data

PierreRambaud commented:
As @mickaelandrieu said, there is no commit from you @peladillas. Can you please create a new pull request with the patch you wanted to submit?
Don't hesitate to contact us on gitter if you need some help ;)

Regards
